packages/react-instantsearch/src/widgets/Range.js

```diff
@@ -1,4 +1,66 @@
 import connectRange from '../connectors/connectRange.js';
-import RangeComponent from '../components/Range.js';
 
-export default connectRange(RangeComponent);
+/**
+ * Since a lot of sliders already exists, we did not include one by default.
+ * However you can easily connect to react-instantsearch an existing one
+ * using the [connectRange connector](/HOC/Range.html).
+ *
+ * @name RangeSlider
+ * @kind component
+ * @category widget
+ * @example
+ *
+ * //Here's an example showing how to connect the airbnb rheostat slider to react-instantsearch using the
+ * //range connector
+ *
+ * import React, {PropTypes} from 'react';
+ * import {connectRange} from 'react-instantsearch/connectors';
+ * import Rheostat from 'rheostat';
+ *
+ * class AirbnbRheostat extends React.Component {
+
+  static propTypes = {
+    min: PropTypes.number.isRequired,
+    max: PropTypes.number.isRequired,
+    value: PropTypes.shape({
+      min: PropTypes.number,
+      max: PropTypes.number,
+    }).isRequired,
+    refine: PropTypes.func.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.updateValue = this.updateValue.bind(this);
+  }
+
+  updateValue(sliderState) {
+    if (sliderState.values[0] !== this.props.min || sliderState.values[1] !== this.props.max) {
+      this.props.refine({min: sliderState.values[0], max: sliderState.values[1]});
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <Rheostat
+          min={this.props.min}
+          max={this.props.max}
+          values={[this.props.value.min, this.props.value.max]}
+          onChange={this.updateValue}
+        />
+        <ol>
+          <lh>Values</lh>
+          <li> {this.props.value.min}
+          </li>
+          <li> {this.props.value.max}
+          </li>
+        </ol>
+      </div>
+    );
+  }
+}
+
+ const AirbnbRheostatConnected = connectRange(AirbnbRheostat);
+ */
+export default connectRange(null);
```

Comment:
Should we have a console.error() that says "We do not provide a Slider, see documentation for how to do it easily" something like that?